### PR TITLE
fix(datastore): bug bash V2 docs (up to Sync Data To Cloud section)

### PR DESCRIPTION
**Pull request by lawmicha**
_Issue #, if available:_

_Description of changes:_

Up to Syncing data to cloud sections

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/lib/datastore/ios/data-access/query-predicate-multiple-snippet.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ let p = Post.keys
 do {
     let result = try await Amplify.DataStore.query(
         Post.self,
-        where: p.rating > 4 && p.status == PostStatus.published
+        where: p.rating > 4 && p.status == PostStatus.active
```
Comment (manaswi223) [inline src/fragments/lib/datastore/ios/data-access/query-predicate-multiple-snippet.mdx:10]: In V2, shouldn't this be .published?

File: src/fragments/lib/datastore/ios/data-access/query-predicate-or-snippet.mdx [hunk 2]
```diff
@@ -7,7 +7,7 @@ let p = Post.keys
 do {
     let result = try await Amplify.DataStore.query(
         Post.self,
-        where: p.rating == nil || p.status == PostStatus.draft
+        where: p.rating == nil || p.status == PostStatus.active
```
Comment (manaswi223) [inline src/fragments/lib/datastore/ios/data-access/query-predicate-or-snippet.mdx:10]: same here. in this case, shouldn't this be `.published`?

File: src/fragments/lib/datastore/ios/data-access/save-snippet.mdx [hunk 3]
```diff
@@ -5,9 +5,9 @@
 ```swift
 do {
     try await Amplify.DataStore.save(
-        Post(title: "My first post", 
-            content: "Amplify.DataStore is awesome!", 
-            status: .draft)
+        Post(title: "My first post",
+             status: .active,
+             content: "Amplify.DataStore is awesome!")
```
Comment (manaswi223) [inline src/fragments/lib/datastore/ios/data-access/save-snippet.mdx:10]: also, Xcode complains about hierarchy order for me:

`_Incorrect argument labels in call (have 'title:status:content:', expected 'id:title:content:createdAt:updatedAt:draft:rating:status:comments:')_`

File: src/fragments/lib/datastore/ios/data-access/query-predicate-or-snippet.mdx (hunk 2 above)
Comment (lawmicha) [reply src/fragments/lib/datastore/ios/data-access/query-predicate-or-snippet.mdx:10]: the schema i used from the first page of the docs is 

```
type Post @model {
  id: ID!
  title: String!
  status: PostStatus!
  rating: Int
  content: String
}

enum PostStatus {
  ACTIVE
  INACTIVE
}
```
is there another PostStatus somewhere that uses .published ?

File: src/fragments/lib/datastore/ios/data-access/save-snippet.mdx (hunk 3 above)
Comment (lawmicha) [reply src/fragments/lib/datastore/ios/data-access/save-snippet.mdx:10]: we're probably using different schema, is it different from http://localhost:3000/lib/datastore/getting-started/q/platform/ios/#sample-schema ?

File: src/fragments/lib/datastore/ios/data-access/query-predicate-multiple-snippet.mdx (hunk 1 above)
Comment (lawmicha) [reply src/fragments/lib/datastore/ios/data-access/query-predicate-multiple-snippet.mdx:10]: i searched for all places for "enum PostStatus" to find the schemas and it appears they are all containing ACTIVE and INACTIVE, can't find which schema generates Published

File: src/fragments/lib/datastore/ios/data-access/query-predicate-or-snippet.mdx (hunk 2 above)
Comment (manaswi223) [reply src/fragments/lib/datastore/ios/data-access/query-predicate-or-snippet.mdx:10]: my mistake, you're correct..I was using a different schema...

**Review by manaswi223 — APPROVED** [review]

**Pull request merged**